transform-plugins/widgets/CSVFormatter-transform.json

```diff
@@ -27,19 +27,19 @@
           "name": "delimiter",
           "widget-attributes": {
             "values": [
+              "CARROT",
```

Comment:
Seems like it was existing code, from https://github.com/caskdata/hydrator-plugins/pull/32.

In fixing this, you should also make a change here:
https://github.com/caskdata/hydrator-plugins/blob/release/1.4/transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVFormatter.java#L90